### ISSUE TITLE
Feature Request, Next suggestion

### ISSUE THREAD

**Issue by bechampion** [zsh-users/zsh-autosuggestions#409]
Wouldn't it be useful to have a `next-suggestion` binding , so it moves up to the current -1 element on your suggestion strategy.

Thanks for zsg-autosuggestions btw!

**Comment by ericfreese** [comment]
There is an open PR #387 that implements part of what you're proposing. I'm not convinced that it is a necessary feature though. Have you tried using the `history-beginning-search-forward` and `history-beginning-search-backward` widgets?

**Comment by ericfreese** [comment]
I'm actually going to close this as a duplicate of #192.

**Comment by benwoodward** [comment]
The problem I'm having with `history-beginning-search-forward` and `history-beginning-search-backward` is that this functionality doesn't make suggestions, it places the whole command in the prompt. Which means that partial selection with `forward-word` isn't possible, instead you have to manually edit the command. I use vim mode which makes this a bit easier, but with complex commands it's still a lot of extra friction.

Basically I'd like to type a word, partial select, skip through history, partial select again, enter command. 

Although, it's not like a dealbreaker, more just a nice-to-have.

**Comment by shqear93** [comment]
https://github.com/zsh-users/zsh-autosuggestions/issues/609#issuecomment-904204583
